pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/FilterParser.java

```diff
@@ -396,4 +387,12 @@ private Operation reverseOp(Operation operation) {
         operatorTranslationMap.put(8, Operation.HDOP_LIKE);
         return operatorTranslationMap;
     }
+
+    static private Map<Integer, LogicalOperation> initLogicalOperatorTransMap() {
+        Map<Integer, LogicalOperation> integerLogicalOperationMap = new HashMap<>();
```

Comment:
I think we'll need to go with an array. I haven't found a way to do a reverse lookup of a Enum based on `int` values. Enum.valueOf() takes a String.